plugins/modules/database/postgresql/postgresql_info.py

```diff
@@ -291,7 +291,7 @@
   returned: if pg_stat_replication view existent
   type: dict
   sample:
-  - { 76580: { "app_name": "standby1", "backend_start": "2019-02-03 00:14:33.908593+03",
+  - { "76580": { "app_name": "standby1", "backend_start": "2019-02-03 00:14:33.908593+03",
```

Comment:
@felixfontein , i think i'll be returned as an integer from a db because of "SELECT r.pid, .." (the `pid` is an integer type).
Will it be converted to string by Ansible?

Reply:
Well, if the module actually tries to return this, it will probably crash. Ansible's (and Python's) JSON encoder dies when encountering non-string keys. It does do some sanitization (datetime objects for example will be handled), but not for dict keys.

Reply:
Anyway, since module return values are always passed through JSON, it is impossible that the module returns this result, no matter whether it crashes or not :)

Reply:
I just checked, it won't crash in this case, but automatically convert the int to a string:
```.py
>>> import json
>>> json.dumps({1: 2})
'{"1": 2}'
```

Reply:
I've just checked with 2.9, it'll be converted
```
    "pending_restart_settings": [],
    "repl_slots": {},
    "replications": {
        "127075": {
            "app_name": "spblnx239",
            "backend_start": "2020-03-26 21:57:44.960738+03",
            "client_addr": "10.44.1.239",
            "client_hostname": "",
            "state": "streaming",
            "usename": "postgres"
        }
    },
```
thanks!